src/install/lockfile.zig

```diff
@@ -1658,15 +1658,13 @@ pub fn saveToDisk(this: *Lockfile, filename: stringZ) void {
         }
     }
 
-    if (comptime Environment.isWindows) {
-        // TODO: make this executable
-        @panic("TODO on Windows");
-    } else {
-        _ = C.fchmod(
-            tmpfile.fd,
-            // chmod 777
-            0o0000010 | 0o0000100 | 0o0000001 | 0o0001000 | 0o0000040 | 0o0000004 | 0o0000002 | 0o0000400 | 0o0000200 | 0o0000020,
-        );
+    switch (bun.sys.fchmod(tmpfile.fd, 0o0000010 | 0o0000100 | 0o0000001 | 0o0001000 | 0o0000040 | 0o0000004 | 0o0000002 | 0o0000400 | 0o0000200 | 0o0000020)) {
```

Comment:
please add comment from before
`// chmod 777`
maybe we ought to just inline it `0o1777`